Renode/startup.resc

```diff
@@ -0,0 +1,51 @@
+$bin?=$ORIGIN/../Objects/controls-leader.elf
+$platform=$ORIGIN/stm32f413.repl
+
+EnsureTypeIsLoaded "Antmicro.Renode.Peripherals.DMA.STM32DMA"
+include $ORIGIN/STM32DMA_Fix.cs
+
+EnsureTypeIsLoaded "Antmicro.Renode.Peripherals.Analog.STM32_ADC"
+include $ORIGIN/STM32_ADC_Fix.cs
+
+EnsureTypeIsLoaded "Antmicro.Renode.Peripherals.UART.STM32_UART"
+include $ORIGIN/STM32_UART_Fix.cs
+
+macro reset
+"""
+    sysbus LoadELF $bin
+"""
+
+macro show_uart_leader
+"""
+    emulation CreateServerSocketTerminal $uart2_telnet "ldr-uart-con" false
+    connector Connect sysbus.usart2 ldr-uart-con
+"""
+
+macro show_uart_display
+"""
+    emulation CreateServerSocketTerminal $uart3_telnet "disp-uart-con" false
+    connector Connect sysbus.usart3 disp-uart-con
```

Comment:
### _**ADD STRINGS TO DISPLAY WHAT PORTS THEY ARE DISPLAYED ON**_

Reply:
too hard.